src/learning-header/AuthenticatedUserDropdown.jsx

```diff
@@ -1,50 +1,61 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
```

Comment:
Were these changes made automatically by your editor? EdX JS styleguide says to prefer single quotes for strings, and we usually try to stick to the edX styles.

Reply:
You're right. My bad. I'll revert those.